src/utils/base64.js

```diff
@@ -71,6 +76,16 @@ export function base64UrlEncodeFromString(str) {
   return base64UrlEncodeFromBytes(bytes);
 }
 
+/**
+ * Decode a base64url encoded string by `base64UrlEncodeFromString`
+ * @param {string} str
+ * @return {string}
+ */
+export function base64UrlDecodeFromString(str) {
```

Comment:
pls add some tests

Reply:
test added. Note I wanted to improve the test format by introducing a testCases array with the encode and decode value. (like the test-linker) But then I realize it is difficult to debug with that because without a description, it's very difficult to find out which pair leads to the error. So I simply copied the test from `base64UrlEncodeFromString`